ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/turnstile/v1/turnstile/clientlibs/site/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[core.forms.components.turnstile.v1.runtime]"
+    dependencies="[core.forms.components.runtime.base,core.forms.components.container.v2.runtime]"/>
```

Comment:
Please get rid of container.v2.runtime, its not required anymore

Reply:
Done